.rosinstall

```diff
@@ -1,19 +1,35 @@
+# Drivers for interfacing with GPS's that give us nmea sentences (like the swift GPS)
```

Comment:
Thanks for the description, but these get nuked when using the CLI commands to set/update repositories.
i.e. running `wstool set src/external_pkgs/some_pkg --git https://github.com/abc/def.git` will basically reset the `.rosinstall` to only contain the entries themselves. Could make it a req to only update these by hand, but that's a little weirdly restrictive :/